etcdserver/api/rafthttp/stream.go

```diff
@@ -214,6 +216,12 @@ func (cw *streamWriter) run() {
 					batched++
 				}
 
+				// TODO: track other messages?
+				// snapshot sends are tracked via separate metrics
+				// heartbeats are tracked via prober https://github.com/coreos/etcd/pull/10022
+				if m.Type == raftpb.MsgProp || m.Type == raftpb.MsgApp || m.Type == raftpb.MsgAppResp {
+					raftSendSeconds.WithLabelValues(m.Type.String(), types.ID(m.To).String()).Observe(took.Seconds())
+				}
```

Comment:
Is there a reason that MsgProp, MsgApp and MsgAppResp are chosen here? I was reading [the doc](https://github.com/etcd-io/etcd/blob/e8b940f268a80c14f7082589f60cbfd3de531d12/raft/doc.go#L197), and you explained snapshot and heartbeats, how about other message types?